release-notes/1.19.0.md

```diff
@@ -0,0 +1,69 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud < release-version >.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* The AMS API documentation created using OpenAPI specification is available on the AMS host at `/api/1.0/swagger.json`.
+* Pagination and filtering support is enabled for containers and applications in the AMS API. This has made several dashboard enhancements possible. You can also use it if you want to interact directly with the AMS using API calls.
```

Comment:
```suggestion
* Pagination and filtering support is enabled for containers and applications in the AMS API. You can use it by directly interacting with AMS using API calls.
```

Not sure if we need to mention something that is internal in release notes? We can mention this in the pulse report though.